.cycloid.yml

```diff
@@ -1,18 +1,30 @@
 # Configuration of the CI pipeline
-pipeline:
+name: 'Magento'
+canonical: 'stack-magento'
+description: 'This stack will deploy a Magento on X Amazon EC2 instances behind an ELB load balancer, using RDS database and ElasticCache.'
+keywords:
+  - 'stack'
+  - 'public'
```

Comment:
I think we can remove the `public` as it already has the `status` attribute on the DB for that.